testdata/Dockerfile

```diff
@@ -10,10 +19,14 @@ RUN go mod verify
 
 COPY . .
 
+ADD https://github.com/calyptia/cmetrics/releases/download/${CMETRICS_RELEASE}/cmetrics_${CMETRICS_VERSION}_${PACKAGEARCH}-headers.deb external/
+ADD https://github.com/calyptia/cmetrics/releases/download/${CMETRICS_RELEASE}/cmetrics_${CMETRICS_VERSION}_${PACKAGEARCH}.deb external/
+RUN dpkg -i external/*.deb
+
 RUN go build -trimpath -buildmode c-shared -o ./bin/go-test-input-plugin.so ./testdata/input/input.go
 RUN go build -trimpath -buildmode c-shared -o ./bin/go-test-output-plugin.so ./testdata/output/output.go
 
-FROM ghcr.io/calyptia/calyptia-fluent-bit/advanced:main
+FROM ghcr.io/calyptia/enterprise/advanced:pr-123
```

Comment:
we should remove this before merging.

Reply:
Yes, just for testing purpose. I'm waiting for https://github.com/calyptia/calyptia-fluent-bit/pull/123 patch and published advanced image.